src/model/resources/storage/storage_all.jl

```diff
@@ -32,6 +33,8 @@ function storage_all!(EP::Model, inputs::Dict, setup::Dict)
 	# Energy withdrawn from grid by resource "y" at hour "t" [MWh] on zone "z"
 	@variable(EP, vCHARGE[y in STOR_ALL, t=1:T] >= 0);
 
+	virtual_discharge_cost=setup["VirtualChargeDischargeCost"]./scale_factor
```

Comment:
It's unclear why `./` is used since both the parameter and `scale_factor` are scalars.